docs/pages/setup/guides/docker.mdx

```diff
@@ -34,6 +34,7 @@ $ docker version
 </TabItem>
 <TabItem scope={["cloud", "enterprise"]} label="Commercial">
 
+- Access to the [customer portal](https://dashboard.gravitational.com/web/login) for the Teleport license
```

Comment:
Would this instruction be applicable to Teleport Cloud? We might want to include a separate `TabItem` for "Teleport Cloud" here. We could also move the edition warning from the top of the page to the "Teleport Cloud" `TabItem` (and replace "Node services" with "resource services")—if Cloud users won't be adding a license file to the Auth Service config, they can use the OSS version to see how the Auth/Proxy work on a local container.